packages/mds-api-server/index.ts

```diff
@@ -8,9 +8,8 @@ import { AuthorizationHeaderApiAuthorizer, ApiAuthorizer, AuthorizerClaims } fro
 import { Params, ParamsDictionary } from 'express-serve-static-core'
 
 export type ApiRequest<P extends Params = ParamsDictionary> = express.Request<P>
-export type ApiQuery<Q extends string> = { query: { [P in Q]: string | string[] } }
 
-export type ApiQuery<Q extends string> = { query: Partial<{ [P in Q]: string }> }
+export type ApiQuery<Q extends string> = { query: { [P in Q]: string | string[] } }
```

Comment:
This should actually be `{ query: Partial<{ [P in Q]: string | string[] }> }`

Reply:
Addressed in the commit after this note